src/utils/index.js

```diff
@@ -114,23 +114,33 @@ export const getReleases = (forcedRelease, inculdedReleases) =>
       label: releaseLabel,
     }));
 
-export const createLink = ({ pathname, linkText, history, navigate }) => {
+export const createLink = ({
+  pathname,
+  linkText,
+  history,
+  navigate,
+  target,
+}) => {
   if (history) {
     return (
       <Button
+        component="a"
+        target={target}
         variant="link"
         target-href={pathname}
-        onClick={() => history.push(pathname)}
+        href={pathname}
       >
         {linkText}
       </Button>
     );
   } else if (navigate) {
     return (
       <Button
+        component="a"
         variant="link"
+        target={target}
         target-href={pathname}
-        onClick={() => navigate(pathname)}
+        href={pathname}
```

Comment:
if we are not using history and navigate why we need this if else block and other changes in this PR concerning passing history and navigate ?

Reply:
so it should be simply 
```js
 return (
    <Button
      component="a"
      variant="link"
      target={target}
      target-href={pathname}
      href={pathname}
    >
      {linkText}
    </Button>
  );
```

Reply:
yes, you correct, we dont need those params